app/gimletlet/app-mgmt.toml

```diff
@@ -25,7 +25,7 @@ request_reset = ["hiffy"]
 name = "drv-stm32xx-sys"
 features = ["h753"]
 priority = 1
-max-sizes = {flash = 2048, ram = 1024}
+max-sizes = {flash = 2048, ram = 2048}
```

Comment:
what happened here?

Reply:
I suspect `app-mgmt.toml` was broken, because it's not checked by CI

Reply:
We apparently don't actually build this task on CI, so it had, yet again, bitrotted. Since I had modified the Gimletlet `net` tasks, I wanted to make sure I hadn't made the `net` task too big for any of the Gimletlet apps, and had to fix this in order to verify that.

Happy to back out the change, because it is, admittedly, unrelated to the diff.